src/common/async-job-runner.js

```diff
@@ -64,6 +64,7 @@ export class AsyncJobRunner extends StepAudit {
     const {
       type, jobId, auditContext = {},
     } = message;
+    log.info(`inside async runner Message: ${JSON.stringify(message)}, stepNames: ${JSON.stringify(stepNames)}`);
```

Comment:
@anagarwa , Please make sure to remove `${JSON.stringify(message)}` from logs after debugging.